README.md

```diff
@@ -1,6 +1,6 @@
 # swift-android-toolchain
 
-Build Swift for Android from Mac and Linux
+Build Swift for Android from Mac
```

Comment:
you think this isn't clear enough? @ephemer

Reply:
Didn't see this sorry, yeah that's enough for the macOS only change!